### ISSUE TITLE
bug: CSS loading issues behind reverse proxies

### ISSUE THREAD

**Issue by icsy7867** [uxbox/uxbox#4431]
### Steps To Reproduce

Use an nginx reverse proxy, deploy penpot using the docker compose file.

### Expected behavior

To have fonts and other CSS items work correctly.  

### Actual behavior

Resizing and fonts dont seem to work correctly.  Checking the chrome dev I am seeing 502 errors with accessing several of the CSS elements.  Not sure if there is anything special about using an nginx proxy to supply HTTPS over the built in traefik intance.  Since i have multiple containers already using the nginx proxy over 443 and 80, I dont think I could easily switch.

Is there are other proxy magic I would need to change?

### Screenshots or video

<img width="623" alt="image" src="https://github.com/penpot/penpot/assets/3788162/f2ac6480-766e-463c-8cd7-f1eb19bf1bf5">

### Desktop (please complete the following information)

Macos, windows chrome and firefox.

### Environment (please complete the following information)

Docker-compose deployment on a linux server.

**Comment by icsy7867** [comment]
had a bit more time and did some digging... It looks like the built in nginx proxy is not forwarding the font requests to the proper destination.  Looking here:

https://github.com/penpot/penpot/blob/12c34c6d42df186f621374a093cd86cd356271d2/docker/images/files/nginx.conf#L140

You can see:

```
            location ~ ^/internal/gfonts/css {
                proxy_pass https://fonts.googleapis.com/css?$args;
                proxy_hide_header Access-Control-Allow-Origin;
                proxy_hide_header Cross-Origin-Resource-Policy;
                proxy_hide_header Link;
                proxy_hide_header Alt-Svc;
                proxy_hide_header Cache-Control;
                proxy_hide_header Expires;

                proxy_ignore_headers Set-Cookie Vary Cache-Control Expires;

                proxy_set_header User-Agent "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/113.0.0.0 Safari/537.36";
                proxy_set_header Host "fonts.googleapis.com";
                proxy_set_header Accept "*/*";

                proxy_cache penpot;

                add_header Access-Control-Allow-Origin $http_origin;
                add_header Cache-Control max-age=86400;
                add_header X-Cache-Status $upstream_cache_status;
            }
```

So if I manually do this the URL switch, I can see that:

https://fonts.googleapis.com/css?family=Ysabeau%20Infant:100,200,300,regular,500,600,700,800,900,100italic,200italic,300italic,italic,500italic,600italic,700italic,800italic,900italic&display=block

Loads successfully.  But for some reason, it is not doing this reverse proxy correctly inside the container.

**Comment by icsy7867** [comment]
HAH! I figured this one out... So the issue was not so much the proxy as it was that I am using podman.  Looking at the nginx file, the nginx resolver was defaulting to the docker value: 172.0.0.11, however this does not seem to work with my podman deployment.

I copied the nginx.conf file out, change this to my podman gateway IP address, and mounted the file as read only in the frontend! This worked...

```
- /ContainerData/penpot/nginx.conf:/etc/nginx/nginx.conf:z,ro
```

Looking even more at the nginx file I see
resolver $PENPOT_INTERNAL_RESOLVER;

So I should be able to set this with an ENV variable.
